addons/website_livechat/static/src/web/discuss_patch.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Discuss" t-inherit-mode="extension">
+        <xpath expr="//div[hasclass('o-mail-Discuss-threadAvatar')]" position="before">
+            <div title="Avatar" t-if="thread.correspondent" class="me-2 d-flex justify-content-center">
+                <img class="rounded o-website_livechat-Visitor-avatar o_object_fit_cover" t-att-src="thread.correspondent.persona.avatarUrl" alt="Avatar"/>
+            </div>
+        </xpath>
```

Comment:
It display multiple avatars for threads which were already displaying threadAvatar.